jcstress-core/src/main/java/org/openjdk/jcstress/infra/processors/JCStressTestProcessor.java

```diff
@@ -676,6 +676,9 @@ private boolean allFieldsAreDefault(TypeElement el) {
                 if (t.getInitializer() != null) return false;
             }
             if (member.getKind() == Tree.Kind.BLOCK) {
+                BlockTree b = (BlockTree)member;
+                // static initializers are fine
+                if (b.isStatic()) continue;
                 // no instance initializers of any kind
                 return false;
```

Comment:
I would write it as:

```
if (member.getKind() == Tree.Kind.BLOCK) {
    BlockTree b = (BlockTree)member;
    // no instance initializers of any kind	                
    if (!b.isStatic()) return false;
}
```

...to match the style of the method.

Reply:
Sure. Rewritten as requested.